lib/msal-common/src/logger/Logger.ts

```diff
@@ -51,12 +50,21 @@ export class Logger {
     // Callback to send messages to.
     private localCallback: ILoggerCallback;
 
-    constructor(loggerOptions: LoggerOptions) {
+    // Package name implementing this logger
+    private packageName: string;
+
+    // Package version implementing this logger
+    private packageVersion: string;
+
+    constructor(loggerOptions: LoggerOptions, packageName: string, packageVersion: string) {
```

Comment:
I think we need to `packageName` and `packageVersion` optional, to prevent breaking if an app picks up the new version of common but not the new version of browser/node.

Reply:
yep, good call. Changed.